indigo-frontend/src/core/components/common/accordion/accordion.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+import {CdkAccordionModule} from '@angular/cdk/accordion';
+
+@Component({
+  selector: 'app-accordion',
+  imports: [CdkAccordionModule],
+  templateUrl: './accordion.component.html'
+})
+export class AccordionComponent {
+  items = [{title: 'Item 1', description: '<table><thead><tr><th>Header 1</th><th>Header 2</th><th>Header 3</th></tr></thead><tbody><tr><td>Data 1</td><td>Data 2</td><td>Data 3</td></tr></tbody></table>'},
```

Comment:
No need to put items here. As this is single accordion component, we can rely only on title as angular input here and content which will be passed to this accordion.